src/plugins/tagfilter/tagfilter.js

```diff
@@ -253,12 +262,10 @@ $document.on( "change", selectorCtrl, function( event )  {
 
 	// Update list of visible items
 	update( elm );
-
-	$elm.trigger( "wb-contentupdated" );
 } );
 
 // Reinitialize tagfilter if content on the page has been updated
-$document.on( "wb-contentupdated", selector, function() {
+$document.on( "wb-contentupdated", selector, function()  {
```

Comment:
There is an extra space

```suggestion
$document.on( "wb-contentupdated", selector, function() {
```